app/aboutUs.html

```diff
@@ -15,6 +15,7 @@
           $("#header").load("header.html"); 
           });
       </script> 
```

Comment:
this currently has merge conflcts because it doesn't include loading footer.html. Check exactly what merge conflicts you have that the Git is suggesting and fix the code accordingly.